### DISCUSSION TITLE
How to downgrade version without new registration

### DISCUSSION THREAD

**Discussion by androbey** [AsamK/signal-cli#776] [Q&A]
Hi all, 

I am currently using signal-cli version 0.9.0 on a Raspberry Pi 4 (2GB) in daemon mode. Unfortunately it causes some problems, which lead to not sending messages anymore (I am sending messages via dbus interface). Time outs and loosing websocket connections appear. 

On a different Raspberry Pi 4 (2GB) I am using version 0.8.1 which works without any of this problems. I now want to downgrade from version 0.9.0 to version 0.8.1 on the other pi. However, it seems that the "config" file is no longer compatible. 
Is there any way to manually adapt the config file to avoid re-registration? Or in other words, is there any way to safely downgrade signal-cli version?

**Comment by 0815-4711** [comment]
I also wonder if this was possible. Might be a workaround for my issue  #1289
